platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -73,7 +73,24 @@ For Wistia, contact Gradle's Video Team.
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="build-authoring"></a>
+### Build authoring improvements
 
+#### Introduce `AttributeContainer#named`
+
+This release introduces a new API on `AttributeContainer` allowing to create attribute values using the new `named` to not require depending on an ObjectFactory.
+
+Consider the following example demonstrating the new API's behavior:
```

Comment:
```suggestion
This makes attribute assignment more concise while preserving the same semantics as creating a named value via `ObjectFactory`:
```

Reply:
There are no spaces before the dash, is it expected?